### DISCUSSION TITLE
Repo Name OR Files

### DISCUSSION THREAD

**Discussion by juliankasimir** [eviden-parallel/community#14] [General]
Likely, we extend the stylelint config of the other repo [stylelint-config](https://github.com/eviden-parallel/stylelint-config) like:

[.stylelintrc](https://github.com/eviden-parallel/parallel/blob/main/.stylelintrc) -> extends -> [.stylelintrc](https://github.com/eviden-parallel/stylelint-config/blob/main/.stylelintrc)

But, this repo contains various config files like Prettier.
If it's also a repo for Prettier, then we could get the [Prettier Sharing configs](https://prettier.io/docs/en/configuration.html#sharing-configurations) from there too.

If the Repo only for stylelint, we may can remove the other files ?
Else we could name it like `format-lint-config`

In [.github](https://github.com/eviden-parallel/.github) Repo, there is also a prettier Config.

**Comment by Th3S4mur41** [comment]
The [stylelint-config](https://github.com/eviden-parallel/stylelint-config) repo contains a prettier configuration for its own internal formatting, so it should not be deleted.
If we'd want a shared prettier config, this would need to happen through a dedicated repository (or at least package => monorepo).

Prettier does not only format html, js and css but also markdown, json and yaml. So there is also a need for it in repos with no code like [.github](https://github.com/eviden-parallel/.github)
